easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.9.2'
+
+homepage = 'http://broadinstitute.github.io/picard/'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM
+  (http://samtools.github.io/hts-specs) format."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/%(name)s/releases/download/%(version)s/']
+sources = ['%(name)s.jar']
```

Comment:
Recent sources aren't versioned anymore?! :(

We should rename the jar to include the version, and add a comment to explain that the renaming should be done manually (until we have proper support to do this automagically in framework).

At the very least, we should include a checksum.

See also https://github.com/hpcugent/easybuild-framework/issues/455#issuecomment-300255441, cc @pneerincx